wand-watcher.lic

```diff
@@ -12,12 +12,32 @@ class WandWatcher
   include DRCI
 
   def initialize
+
+    arg_definitions = [
+      [
+        {
+          name: 'delay', 
+          regex: /^\d+$/, 
+          optional: true, 
+          description: 'Override wand_watcher_startup_delay from yaml' 
+        },
+        {
+          name: 'reset', 
```

Comment:
What’s a use case that someone would want/need to do this? Just curious if this is something I may need to add to my routine or not. Thanks

Reply:
If timers get out of sync for some reason.